src/main/plugins/com.sophos.tocjs/integrator.xml

```diff
@@ -29,7 +32,7 @@ See the accompanying LICENSE file for applicable license.
             <isset property="output.file" />
         </not>
     </condition>
-    <property name="output.file" value="${output.dir}${file.separator}toctree.js"/>
+    <property name="output.file" value="${ditaot.render.dir}${file.separator}toctree.js"/>
```

Comment:
Along with this - I think it's best to change the override property from `temp.render.dir.name` to `temp.output.dir.name`.